src/app/config.ts

```diff
@@ -1 +1,2 @@
 export const BACKEND_V1_URL = "http://localhost:3000/api/v1";
+export const BACKEND_V2_URL = "https://ucab-profile-backend.playoffside.online/api/v1";
```

Comment:
AMBAS son APIv1
```suggestion
export const BACKEND_V1_URL = "https://ucab-profile-backend.playoffside.online/api/v1";
```
Si emmanuel hiciera un nuevo versionamiento como `https://ucab-profile-backend.playoffside.online/api/v2` entonces si te creeria que amerita un `API_V2` pero este no es el caso.]

Lejos de eso... esto deberia evolucionar a algo asi:
```suggestion
export const BACKEND_V1_URL = process.env.REACT_APP_BLABLABLA_BACKEND_V1_URL;
```
Pero supongo aun no se hace el tema del .env

Reply:
Listo los cambios a todas las URL de los servicios